ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpSourceSpec.scala

```diff
@@ -3,10 +3,14 @@
  */
 package akka.stream.alpakka.ftp
 
+import java.nio.file.attribute.PosixFilePermission
+
 import akka.stream.IOResult
 import akka.stream.scaladsl.{Keep, Sink}
 import akka.stream.testkit.scaladsl.TestSink
 import org.scalatest.time.{Millis, Seconds, Span}
+
+import scala.concurrent.duration._
```

Comment:
this is to have the `.millis` and `.minute` conversions from `duration` package object. would you prefer not to use them?